setup.cfg

```
@@ -0,0 +1,54 @@
+# Minimum target setuptools 39.2.0
+
+[metadata]
+name = ansible-core
+version = attr: lib.ansible.release.__version__
+description = Radically simple IT automation
+long_description = file: README.rst
+author = Ansible, Inc.
+author_email = info@ansible.com
+url = https://ansible.com/
+project_urls =
+    Bug Tracker=https://github.com/ansible/ansible/issues
+    CI: Azure Pipelines=https://dev.azure.com/ansible/ansible/
+    Code of Conduct=https://docs.ansible.com/ansible/latest/community/code_of_conduct.html
+    Documentation=https://docs.ansible.com/ansible-core/
+    Mailing lists=https://docs.ansible.com/ansible/latest/community/communication.html#mailing-list-information
+    Source Code=https://github.com/ansible/ansible
+license = GPLv3+
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Environment :: Console
+    Intended Audience :: Developers
+    Intended Audience :: Information Technology
+    Intended Audience :: System Administrators
+    License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)
+    Natural Language :: English
+    Operating System :: POSIX
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+    Programming Language :: Python :: 3 :: Only
+    Topic :: System :: Installation/Setup
+    Topic :: System :: Systems Administration
+    Topic :: Utilities
+
+[options]
+python_requires = >=3.8
```

I'm pretty sure this option can be moved here as well
```suggestion
package_dir =
    = lib
    ansible_test = test/lib/ansible_test
python_requires = >=3.8
```

---

Since we require `packages` in `setup.py`, I preferred to just keep them together for clarity.